manual/src/ornate/cookbook.md

```diff
@@ -325,3 +325,32 @@ You can use it e.g. with `[sbt-native-packager](https://github.com/sbt/sbt-nativ
 ~~~ scala
 mappings in (Compile, packageBin) ++= (webpack in (Compile, fullOptJS)).value.map { f => f.data -> f.data.getName() },
 ~~~
+
+## How to package for [Electron](https://electronjs.org)
+
+Packaging for Electron is actually not that hard either (but some more manual steps are needed to fully
+get this to work - e.g., writing Facades for Electron's built-in types.)
+
+### Add an extra Webpack configuration to your project
+
+A suggested location is `src/main/resources`. Create a file named `electron.webpack.config.js` there and add these lines:
+
+~~~ javascript
+module.exports = require('./scalajs.webpack.config');
+module.exports.target = 'electron';
+~~~
+
+Now, add to your `build.sbt`:
+
+~~~ scala
+webpackConfigFile :=
+Some(baseDirectory.value / "src" / "main" / "resources" / "electron.webpack.config.js")
+~~~
+
+**DON'T** add any `electron` dependencies to your `npmDependencies`!
+
+When you now run `fastOptJS` or `fullOptJS`, you get an appropriate webpack configuration
+and the generated JS files will be packaged correctly.
+
+You still have to add the generated JavaScript file to the `package.json` manually
+to have it started automatically. You could also create a build task that does this for you.
```

Comment:
Can you give an example of how to do that? Also, if you add a test project you could also add a build task that does it.

Reply:
Any advice about the Electron’s facades to write? Are some of them available somewhere?